resources/js/Pages/Dashboard/HR/JobOpenings/Fulfilled.vue

```diff
@@ -0,0 +1,188 @@
+<style lang="scss" scoped>
+.job-opening-item:first-child:hover {
+  border-top-left-radius: 10px;
+  border-top-right-radius: 10px;
+}
+
+.job-opening-item:last-child {
+  border-bottom: 0;
+
+  &:hover {
+    border-bottom-left-radius: 10px;
+    border-bottom-right-radius: 10px;
+  }
+}
+
+.reference-number {
+  padding: 2px 6px;
+  border-radius: 6px;
+  top: -4px;
+  background-color: #ddf4ff;
+  color: #0969da;
+}
+
+.sidebar {
+  svg {
+    width: 20px;
+    top: 3px;
+    color: #9da3ae;
+  }
+
+  span {
+    top: -2px;
+  }
+
+  .active {
+    background-color: #e5e7ea;
+
+    svg {
+      color: #6c727f;
+    }
+
+    span {
+      color: #121826;
+    }
+  }
+}
+
+.dot {
+  height: 11px;
+  width: 11px;
+  top: 3px;
+}
+
+.active {
+  .dot {
+    background-color: #56bb76;
+  }
+}
+
+.inactive {
+  .dot {
+    background-color: #c8d7cd;
+  }
+}
+</style>
+
+<template>
+  <layout :notifications="notifications">
+    <div class="ph2 ph5-ns">
+      <!-- BREADCRUMB -->
+      <div class="mt4-l mt1 mb4 mw6 br3 center breadcrumb relative z-0 f6 pb2">
+        <ul class="list ph0 tc-l tl">
+          <li class="di">
+            <inertia-link :href="'/' + $page.props.auth.company.id + '/company'">{{ $t('app.breadcrumb_company') }}</inertia-link>
+          </li>
+          <li class="di">
+            <inertia-link :href="'/' + $page.props.auth.company.id + '/dashboard/hr'">{{ $t('app.breadcrumb_hr') }}</inertia-link>
+          </li>
+          <li class="di">
+            {{ $t('app.breadcrumb_hr_job_openings_active') }}
+          </li>
+        </ul>
+      </div>
```

Comment:
You can now use the new breadcrumb template
```suggestion
      <breadcrumb
                  :previous-url="'/' + $page.props.auth.company.id + '/dashboard/hr'"
                  :previous="$t('app.breadcrumb_hr')"
      >
        {{ $t('app.breadcrumb_hr_job_openings_active') }}
      </breadcrumb>
```